go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/go-playground/validator/v10 v10.3.0
 	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/google/uuid v1.1.0
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/testify v1.4.0
```

Comment:
is it necessary to upgrade here?
edgex-go is using v1.5.1.  should we make them sync?
https://github.com/edgexfoundry/edgex-go/blob/8261b4a991e517cb3be3346222253ea56d6b87ec/go.mod#L23

Reply:
go test did this automatically... I am good with moving it to v1.5.1, Done!